doc/api/models/schemas/NextDagPrice.yml

```diff
@@ -0,0 +1,12 @@
+type: object
+description: Information about the next DAG price and when it will take effect
+properties:
+  price:
+    type: integer
+    format: int64
+    description: The next DAG price (in the same format as used internally in configuration)
```

Comment:
Since this is user facing API docs, I would explicitly describe the format. I think we decided on datum/USD?

Reply:
This is now changed to DAG/USD but as a fraction, we need the doc here to correspond to whatever fractional basis format that we're planning to return here (Or whatever it's presently represented as.)